### PR TITLE
Adding vertical alignment to font-setting

### PR DESCRIPTION
**Pull request by stulees**
- Optionally include `vertical-alignment` as an attribute to `font-setting` to include vertical alignment toolbar buttons to save vertical alignment configuration to `fontData`
- e2e test coverage added
- Feature version bump

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: gulpfile.js [hunk 1]
```diff
@@ -141,7 +141,7 @@
 
   gulp.task("e2e:test", factory.testE2E());
   gulp.task("e2e:test-ng", ["webdriver_update"], factory.testE2EAngular({
-    src: ["test/e2e/angular/*test-ng.js"]
+    src: ["test/e2e/angular/font-setting-test-ng.js"]
```
Comment (donnapep) [inline gulpfile.js]: @stulees This will need to be changed back.

File: gulpfile.js [hunk 2]
```diff
@@ -174,7 +174,7 @@
   ));
 
   gulp.task("test", function(cb) {
-    runSequence("build", "test:unit", "e2e:server", "e2e:test", "e2e:test-ng", "e2e:server-close", cb);
+    runSequence("build", /*"test:unit", */"e2e:server", /*"e2e:test", */"e2e:test-ng", "e2e:server-close", cb);
```
Comment (donnapep) [inline gulpfile.js]: @stulees And this.

File: src/_angular/font-setting/dtv-font-setting.js [hunk 3]
```diff
@@ -158,8 +160,14 @@
                */
               skin_url: "//s3.amazonaws.com/rise-common/styles/tinymce/rise",
               statusbar: false,
-              toolbar: "fontselect fontsizeselect | alignleft aligncenter alignright alignjustify | forecolor backcolor | bold italic underline",
+              toolbar: "fontselect fontsizeselect | alignleft aligncenter alignright alignjustify" +
+              ($scope.verticalAlign ? " aligntop alignmiddle alignbottom" : "") +
```
Comment (donnapep) [inline src/_angular/font-setting/dtv-font-setting.js]: @stulees Is vertical alignment supposed to be an optional setting?

File: src/_angular/font-setting/dtv-font-setting.js [hunk 4]
```diff
@@ -383,6 +410,51 @@
               sheet.addRule("@font-face", rule);
             }
           }
+
+          function addVerticalAlignButtons(editor) {
+            editor.addButton("aligntop", {
+              image: "//s3.amazonaws.com/Rise-Images/Icons/align-top.svg",
```
Comment (donnapep) [inline src/_angular/font-setting/dtv-font-setting.js]: @stulees Can this be part of the repo and added to the `dist` folder as part of the build?

File: gulpfile.js (hunk 1 above)
Comment (stulees) [inline gulpfile.js]: @donnapep Good catch, mean to uncomment that. Thanks

File: src/_angular/font-setting/dtv-font-setting.js (hunk 3 above)
Comment (stulees) [inline src/_angular/font-setting/dtv-font-setting.js]: @donnapep Yes, only Spreadsheet requires it so far, so this setting is not always applicable. Thanks

File: src/_angular/font-setting/dtv-font-setting.js (hunk 4 above)
Comment (stulees) [inline src/_angular/font-setting/dtv-font-setting.js]: @donnapep They could, but then even when a Widget has this repo as a dependency, the Widget repo gulp file will still require moving the images into its dist/ folder when it builds. Seemed like an unnecessary extra step to me and easier to just host in one spot. What you think?

Comment (donnapep) [inline src/_angular/font-setting/dtv-font-setting.js]: @stulees True. I was just thinking that our repos are already doing that for some files anyway (e.g. copying FontAwesome file and even some images). I guess it's a question of whether we want to be consistent with how we've been doing things thus far, or if this way is better (not sure?), then start doing things this way instead.

Comment (stulees) [inline src/_angular/font-setting/dtv-font-setting.js]: @donnapep This is the first angular directive from this repo that requires an image. In this situation we're vulnerable to an incorrect path of where the image needs to reside and it is easy to forget to include a gulp task in a widget repo to move the image where it needs to be. Personal preference is to just host the image in the bucket.

**Comment by stulees** [comment]
@donnapep I've corrected the gulpfile, thanks.

**Comment by donnapep** [comment]
👍

**Pull request merged**